data/operators.csv

```diff
@@ -320,6 +320,7 @@ Plusnet,ISP,filtering,partially safe,6871,10921
 Millenicom,ISP,,unsafe,34296,10945
 Trustpower,ISP,started,unsafe,55850,11135
 NetCup,cloud,,unsafe,197540,11196
+True Online,ISP,,unsafe,17552,11486
```

Comment:
Can confirm AS17552 does not filter RPKI invalid. But they signed their prefixes on RPKI. Suggest change to `partially safe` category.
```suggestion
True Online,ISP,signed,partially safe,17552,11486
```